controllers/Redirects.php

```diff
@@ -106,14 +108,19 @@ public function index(): void
         }
     }
 
+    public function create()
+    {
+        $this->formLayout = 'sidebar';
+
+        parent::create();
+    }
+
```

Comment:
Why is this required if we're already setting the formLayout to sidebar by default?